config/initializers/deprecations.rb

```diff
@@ -1 +1,29 @@
 ActiveSupport::Deprecation.behavior = :silence if Rails.env.production?
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    unless Foreman.in_rake?
+      if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?('mysql')
```

Comment:
can also reduce indent level:
```suggestion
      break unless ActiveRecord::Base.connection.adapter_name.downcase.starts_with?('mysql')
```